<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Project 1 </title>
  <style>

          .badge {
    background-color: #DA4167;
    color: white;
    padding: 2px 6px;
    border-radius: 4px;
    font-size: 0.75em;
   margin-left: 6px;
      }

    body {
      font-family: Arial, sans-serif;
      max-width: 800px;
      margin: 40px auto;
      padding: 0 20px;
      line-height: 1.6;
      color: #333;
    }
    img {
      width: 100%;
      max-height: auto;
      object-fit: contain;
      border-radius: 8px;
      margin-bottom: 20px;
    }
    h1 {
      font-size: 1.8em;
      margin-bottom: 10px;
    }
    p.description {
      font-size: 0.95em;
      color: #555;
    }
  </style>
</head>
<body>

  <img src="Figure 1 _ scheme.png" alt="Project Banner">

  <span class="badge"><em>Nature comm</em> Journal Submission â†’</span>

  <h1>Uncovering the impact of RNA-Seq preprocessing on AI disease models: A large-scale multicohort study</h1>

  <p class="description">
    Despite advances in transcriptomic technologies, bulk RNA-sequencing (RNA-Seq) remains a key platform, including for liquid biopsy application, due to its comprehensive and stable representation of gene expression. Although AI applications in transcriptomics have expanded, the impact of data preprocessing on model performance and biological interpretation remains largely unexplored. The high dimensionality of RNA-Seq data with the absence of standardized guidelines for normalization, scaling, contributes to model overfitting and poor reproducibility, limiting clinical and biological applications. Here, we systematically evaluated the effects of six normalization methods and three scaling techniques across 13 machine learning (ML) algorithms using 5,852 RNA-seq samples from 25 cohorts. By optimizing 19.3 million sub-models, we demonstrate how preprocessing alters model performance, feature selection, and biomarker identification through distributional shifts affecting feature importance. We validated our findings via an amyotrophic lateral sclerosis (ALS) classification model integrating optimized strategies to overcome conventional limitations. Our study provides a comprehensive benchmarking framework that underscores the critical role of preprocessing in reliable and interpretable AI-based transcriptomic modeling. 
   </p>Keywords: Transcriptomic analysis, Sample normalization and scaling methods, Machine learning, Disease specification   </p>

</body>
</html>